example/src/App.js

```diff
@@ -1,21 +1,29 @@
-import React, { Component } from 'react'
+import React from 'react'
 import ReactTimer from './dist'
 
-export default class App extends Component {
-  render () {
-    return (
-      <div>
-        <ReactTimer
-          start={30}
-          end={0}
-          progression='desc'
-          delta={2}
-          onComplete={(value) => console.log('ENDED WITH VALUE', value)}
-          onTick={(value) => console.log('CURRENT VALUE', value)}
-        >
-          {(time) => <div>{time}</div>}
-        </ReactTimer>
-      </div>
-    )
-  }
+const App = () => {
+  return (
+    <div>
+      <ReactTimer
+        start={30}
+        end={0}
+        delta={2}
+        onComplete={(value) => console.log('ENDED WITH VALUE', value)}
+        onTick={(value) => console.log('CURRENT VALUE', value)}
+      >
+        {(time) => <div>{time}</div>}
+      </ReactTimer>
+      <ReactTimer
+        start={0}
+        end={30}
+        delta={2}
```

Comment:
I think the prop `delta` pretty much becomes unused since we have abstracted the progression into a user-defined callback